### PR TITLE
Restore Open Sans as the default font

### PR DESCRIPTION
**Pull request by jeffswartz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -432,8 +432,7 @@ See the [Phone dial-out](#phone-dial-out) section.
 
 * `USE_GOOGLE_FONTS` (Optional, default value: true): Whether the client app will load
```
Comment (emilianop11) [inline README.md:433]: Please be aware that the USE_GOOGLE_FONTS property is listed twice in the readme. One should go away :)

Comment (emilianop11) [reply README.md:433]: Also, this env tracks down to the `useGoogleFonts` variable, which i see being referenced but not used. Am i missing something?

**Review by emilianop11 — CHANGES_REQUESTED** [review]

File: web/css/fonts.css [hunk 2]
```diff
@@ -1,21 +1,14 @@
-/* spezia-regular - latin */
+/* open-sans-regular - latin */
 @font-face {
-  font-family: 'Spezia';
+  font-family: 'Open Sans';
   font-style: normal;
   font-weight: 400;
   font-display: swap;
```
Comment (Raulkg) [inline web/css/fonts.css:6]: Preload this font in HTML. using the below line which makes the application faster.

`<link rel="preload" as="font" href="/fonts/custom-font.woff2" type="font/woff2" crossorigin="anonymous">`

**Review by Raulkg — APPROVED** [review]
LGTM

**Review by emilianop11 — APPROVED** [review]

**Pull request merged**